covid19_sfbayarea/data/san_mateo/total_deaths.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any, Dict, List, cast
+from .power_bi_querier import PowerBiQuerier
+from .utils import dig
+
+class TotalDeaths(PowerBiQuerier):
```

Comment:
The county sums the race data, so I am using exactly the method they use.

We could do this offline too, but I think it's best to use the same queries so we don't have weird data issues.